packages/docs/page-config/navigationRoutes.ts

```diff
@@ -342,6 +342,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: "sidebar-item",
         displayName: "Sidebar Item",
       },
+      {
```

Comment:
Needs NEW label.

![image](https://github.com/epicmaxco/vuestic-ui/assets/5394573/7d49f655-f10e-4d3c-88ba-7f54a58f226a)

You can check https://github.com/epicmaxco/vuestic-ui/pull/3633